workflow-service/src/main/java/com/redhat/parodos/common/controller/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,55 @@
+package com.redhat.parodos.common.controller;
+
+import java.util.Date;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import com.redhat.parodos.common.exceptions.ResourceAlreadyExistsException;
+import com.redhat.parodos.common.exceptions.ResourceNotFoundException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+@RestControllerAdvice
+public class ControllerExceptionHandler {
```

Comment:
I wouldn't like to reuse the same code between the two service. It is too coupled to the workflow service and as it will expand, more will be added to it which has relevancy to the workflow service itself.

Reply:
make sense, thank you. Let's think about it later